src/main/java/sixgaezzang/sidepeek/projects/service/ProjectService.java

```diff
@@ -124,4 +135,12 @@ private void validateLoginUserIncludeMembers(Long loginId, Project project) {
             .orElseThrow(() -> new InvalidAuthenticationException(ONLY_OWNER_AND_FELLOW_MEMBER_CAN_UPDATE));
     }
 
+    private LocalDate getStartDayOfWeek(LocalDate endWeekDay) {
+        LocalDate startWeekDay = endWeekDay;
+        while (!startWeekDay.getDayOfWeek().equals(MONDAY)) {
+            startWeekDay = startWeekDay.minusDays(1L);
+        }
+        return startWeekDay;
+    }
```

Comment:
금주의 인기 프로젝트의 기간이 **`이번주 월요일 ~ 오늘`** 이군요!
아래와 같이 `TemporalAdjusters`를 사용하면 while문없이 가장 가까운 월요일을 찾을 수 있습니당! 참고 부탁드려용!🙏🏻
```suggestion
    private LocalDate getStartDayOfWeek(LocalDate endWeekDay) {
        return endWeekDay.with(TemporalAdjusters.previousOrSame(DayOfWeek.MONDAY));
    }
```

Reply:
어머 이런건 어떻게 잘 아시는 거죠!! 바로 적용하겠습니당!! 많이 배워갑니다용!!🥰